tasks/manage_app.yml

```diff
@@ -84,7 +84,7 @@
     field_selectors:
       - "status.phase=Running"
   register: pods
-  until: pods.resources is defined and ( pods.resources | length ) == ( expected_running_pods | int )
+  until: pods.resources is defined and ( pods.resources | length ) >= ( expected_running_pods | int )
```

Comment:
we could also add a `when` instruction testing if `deployments | length > 0`

```
when: deployments |  length > 0
```

Reply:
Indeed, I think your solution is more reliable @lunika, but why not using:

```yaml
  when: expected_running_pods | int > 0
```

Reply:
Ok I did this. Checking on `expected_running_pods` is easier to understand as we have this variable at hand here.

Reply:
> Indeed, I think your solution is more reliable @lunika, but why not using:
> 
> ```yaml
>   when: expected_running_pods | int > 0
> ```

Yes, you are a very clever man

![](https://media.giphy.com/media/7XuPYJXaF1CBAmbwQQ/giphy.gif)